packages/rancher-kiali-server/package.yaml

```diff
@@ -1,5 +1,5 @@
 url: https://kiali.org/helm-charts/kiali-server-1.32.0.tgz
-version: 100.0.0
```

Comment:
Same here, can we make this file identical to https://github.com/rancher/charts/blob/dev-v2.5/packages/rancher-kiali-server/package.yaml ?